packages/lit-html/src/directive-helpers.ts

```diff
@@ -67,13 +67,15 @@ export const isTemplateResult = (
 /**
  * Tests if a value is a DirectiveResult.
  */
-export const isDirectiveResult = (
-  value: unknown,
-  klass?: DirectiveClass
-): value is DirectiveResult =>
-  klass === undefined
-    ? (value as DirectiveResult)?._$litDirective$ !== undefined
-    : (value as DirectiveResult)?._$litDirective$ === klass;
+export const isDirectiveResult = (value: unknown): value is DirectiveResult =>
```

Comment:
I'm not sure how useful this is if we add `classForDirectiveResult`... We don't use it at all now.

We could build a more formal branding mechanism in, like an optional `static directiveType = Symbol.for('RenderLight')`, and if you pass the 2nd class argument, it will compare the `directiveType`s for both? (and then get rid of `classForDirectiveResult` and go back to having just `isDirectiveResult`)?

Reply:
Ok, so will leave it